src/05-partial-mocking/index.test.ts

```diff
@@ -11,10 +19,30 @@ describe('partial mocking', () => {
   });
 
   test('mockOne, mockTwo, mockThree should not log into console', () => {
-    // Write your test here
+    const consoleSpy = jest.spyOn(console, 'log');
+
+    mockOne();
+    mockTwo();
+    mockThree();
+
+    expect(consoleSpy).not.toHaveBeenCalledWith('foo');
+    expect(consoleSpy).not.toHaveBeenCalledWith('bar');
+    expect(consoleSpy).not.toHaveBeenCalledWith('baz');
+
```

Comment:
`expect(consoleSpy).not.toHaveBeenCalled();`
would be correctly..

Reply:
It would be mistake, because according the task, functions should be called, but without login to console, so this test case is correct

Reply:
If console.log doesn't be called at all, than doesn't matter with which params it wouldn't.